### PR TITLE
[11.0][MIG] product_state

### PR DESCRIPTION
**Pull request by asaunier**
Simple port of https://github.com/OCA/product-attribute/tree/10.0/product_state to Odoo 11.

### CODE REVIEW

File: product_state/views/product_views.xml
Comment (pedrobaeza) [inline product_state/views/product_views.xml]: Why this extra space?

Comment (asaunier) [reply product_state/views/product_views.xml]: Good catch, thanks :)
Actually it is not an extra space: I had converted a tab to 4 spaces and it seems github does not show them the same way. Your comment made me realize the opening ``<field>`` tag (line 22) was not correctly indented either => I have just pushed 0d332ec making sure the whole file has no tab.

**Review by pedrobaeza — APPROVED** [review]

**Review by p-tombez — APPROVED** [review]
LGTM, didn't review i18n

**Review by tschanzt — APPROVED** [review]

File: product_state/README.rst
Comment (grindtildeath) [inline product_state/README.rst]: 11.0

File: product_state/README.rst
Comment (grindtildeath) [inline product_state/README.rst]: IMO you should get rid of the first part and have something like : 'This module introduces the state field on product template and allows simple product life cycle.'

File: product_state/README.rst
Comment (grindtildeath) [inline product_state/README.rst:41]: Add yourself ?

File: product_state/__manifest__.py
Comment (grindtildeath) [inline product_state/__manifest__.py]: https://github.com/OCA/product-attribute

File: product_state/README.rst
Comment (grindtildeath) [inline product_state/README.rst]: You can remove Installation and Configuration sections.

File: product_state/README.rst
Comment (grindtildeath) [inline product_state/README.rst]: Please update this section considering : https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Review by grindtildeath — COMMENTED** [review]
Out of README and __manifest__ to update :+1:

**Comment by asaunier** [comment]
@grindtildeath Thanks! Updated.

**Review by guewen — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash your last 2 commits before merging.

**Comment by asaunier** [comment]
@pedrobaeza Squashed!
Thanks all for reviewing!

**Pull request merged**
